.github/workflows/test.yml

```diff
@@ -90,12 +106,37 @@ jobs:
       - name: Test
         working-directory: ./frontend
         run: yarn test:ci
+      - name: Save Build Output for Sonar
+        uses: actions/upload-artifact@v2
+        if: success()
+        with:
+          name: yarn-build-output
+          path: frontend/coverage
+          retention-days: 1
+
+  sonar:
+    runs-on: ubuntu-latest
+    needs: [test-java, test-node]
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Disable shallow clones so Sonar can have all the data
+      - name: Retrieve java build
+        uses: actions/download-artifact@v2
+        with:
+          name: java-build-output
+      - name: Retrieve yarn build
+        uses: actions/download-artifact@v2
+        with:
+          name: yarn-build-output
       - name: Sonar analysis
-        if: ${{ github.actor != 'dependabot[bot]' }}
+        working-directory: ./backend
```

Comment:
Are we still running Sonarqube against the frontend by doing this?

Reply:
Yes, the sonar command is a gradle task, so it's always been run from the backend directory—the projectBaseDir property is what makes it include `./frontend` as well.